help/destinations/catalog/email-marketing/mailchimp-interest-categories.md

```diff
@@ -0,0 +1,239 @@
+
+>[!IMPORTANT]
+>
+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+Read [Activate profiles and segments to streaming segment export destinations](/help/destinations/ui/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Mapping considerations and example {#mapping-considerations-example}
+
+To correctly send your audience data from Adobe Experience Platform to the [!DNL Mailchimp Interest Categories] destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination.
+
+To correctly map your XDM fields to the [!DNL Mailchimp Interest Categories] destination fields, follow the steps below:
+
+1. In the **[!UICONTROL Mapping]** step, select **[!UICONTROL Add new mapping]**. You will see a new mapping row on the screen.
+1. In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select attributes]** category and select the XDM attribute or choose the **[!UICONTROL Select identity namespace]** and select an identity.
+1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** and select an identity or choose **[!UICONTROL Select attributes]** category and select from the list of attributes populated from the [!DNL Mailchimp] API. *Note any custom attributes that have been added to the selected [!DNL Mailchimp] Audience will also be available.*
+
+    The default mappings available between your XDM profile schema and [!DNL Mailchimp Interest Categories] will be as below:
+    | Source Field | Target Field | Notes |
+    | --- | --- | --- |
+    |`IdentityMap: Email`|`Identity: email`| Mandatory: Yes |
+    |`xdm: person.name.firstName`|`Attribute: FNAME`| |
+    |`xdm: person.name.lastName`|`Attribute: LNAME`| |
+    |`xdm: person.birthDayAndMonth`|`Attribute: BIRTHDAY`| |
+
+    Additionally, `ADDRESS` is a special target field known as a `merge field` within your [!DNL Mailchimp] audience. The [[!DNL Mailchimp] documentation](https://mailchimp.com/developer/marketing/docs/merge-fields/) defines the required keys as `addr1`, `city`, `state`, and `zip`, and the optional keys `addr2` and `country`. The values for these fields must be strings. If any of the `ADDRESS` field mappings are present, the destination passes the `ADDRESS` object to the [!DNL Mailchimp] API for update. Any `ADDRESS` fields that are not mapped has its value default to `NULL` except for country which defaults to `US`. 
```

Comment:
what does this section mean for me a a user, though? is there any recommendation included here?

Reply:
please ignore, this is well explained by the example in the next section 👍